source/introduction.rst

```diff
@@ -67,6 +67,13 @@ off the list to a next stage is termed the *Sender*. The firmware stage
 receiving the list is termed the *Receiver*. The *Receiver* can update fields in
 any entry in the list and is allowed to remove entries.
 
+The TL should not hold pointers or addresses within its entries. These can make
```

Comment:
I don't understand how **physical addresses** make it difficult to relocate a transfer list.  Nothing would need to change at all on a relocation.

I do understand that if there were offsets or virtual addresses it would be a problem.

Reply:
Stuart's comment (which I agree with) hasn't been addressed in this version. If the TL points to memory outside the TL (which is legitimate) then the address is unaffected by the relocation. If you're concerned about pointers to data within the TL (or at least to the same memory region to be relocated), then this absolutely should be banned. In that case, this specific case should be stated, and the "should" can be changed to "must".